homeassistant/components/ecoforest/sensor.py

```diff
@@ -88,6 +93,58 @@ class EcoforestSensorEntityDescription(
         icon="mdi:alert",
         value_fn=lambda data: data.alarm.value if data.alarm else "none",
     ),
+    EcoforestSensorEntityDescription(
+        key="depression",
+        translation_key="depression",
+        native_unit_of_measurement=UnitOfPressure.PA,
+        device_class=SensorDeviceClass.ATMOSPHERIC_PRESSURE,
+        entity_registry_enabled_default=False,
+        value_fn=lambda data: data.depression,
+    ),
+    EcoforestSensorEntityDescription(
+        key="working_hours",
+        translation_key="working_hours",
+        native_unit_of_measurement=UnitOfTime.HOURS,
+        device_class=SensorDeviceClass.DURATION,
+        entity_registry_enabled_default=False,
+        value_fn=lambda data: data.working_hours,
+    ),
+    EcoforestSensorEntityDescription(
```

Comment:
Maybe you can add `native_unit_of_measurement="ignitions"` as well

Reply:
The ignitions are just a numerical value, is there any unit measure for that?

Reply:
I mean, it's counting the amount of ignitions right, you can put custom values as unit of measurement